src/groupeddataframe/splitapplycombine.jl

```diff
@@ -366,19 +392,16 @@ julia> by(df, :a) do d # do syntax for the slower variant
 │ 3   │ 3     │ 10    │
 │ 4   │ 4     │ 12    │
 
-julia> combine(gd, :c => (x -> 2 .* x) => :y) # specifying a name for target column
-8×2 DataFrame
-│ Row │ a     │ y     │
-│     │ Int64 │ Int64 │
-├─────┼───────┼───────┤
-│ 1   │ 1     │ 2     │
-│ 2   │ 1     │ 10    │
-│ 3   │ 2     │ 4     │
-│ 4   │ 2     │ 12    │
-│ 5   │ 3     │ 6     │
-│ 6   │ 3     │ 14    │
-│ 7   │ 4     │ 8     │
-│ 8   │ 4     │ 16    │
+julia> combine(gd, :c => (x -> sum(log.(x))) => :sum_log_c) # specifying a name for target column
```

Comment:
Just being picky. :-p
```suggestion
julia> combine(gd, :c => (x -> sum(log, x)) => :sum_log_c) # specifying a name for target column
```